RiotSharp/RiotApi.cs

```diff
@@ -212,37 +212,30 @@ public Champion GetChampion(Region region, int championId)
         public Dictionary<long, List<MasteryPage>> GetMasteryPages(Region region, List<long> summonerIds)
```

Comment:
Sounds good to me.  Wasn't sure if we wanted to break backwards compatibility with RiotSharp, but V3 is breaking a lot of stuff anyway.  I will add the changes.